features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin_new/source/src/app/components/AdminOperations/APICategories/EditAPICategory.jsx

```diff
@@ -0,0 +1,290 @@
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState } from 'react';
+import API from 'AppData/api';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import Button from '@material-ui/core/Button';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { Link } from 'react-router-dom';
+import { withStyles } from '@material-ui/core/styles';
+import FormControl from '@material-ui/core/FormControl';
+import Grid from '@material-ui/core/Grid';
+import Icon from '@material-ui/core/Icon';
+import Paper from '@material-ui/core/Paper';
+import Alert from 'AppComponents/Shared/Alert';
+import settings from '../../../../../../site/public/conf/settings';
+
+const styles = (theme) => ({
+    root: {
+        flexGrow: 1,
+        marginTop: 10,
+        display: 'flex',
+        flexDirection: 'column',
+        padding: 20,
+    },
+    titleWrapper: {
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center',
+        marginBottom: theme.spacing(3),
+    },
+    titleLink: {
+        color: theme.palette.primary.main,
+        marginRight: theme.spacing(1),
+    },
+    contentWrapper: {
+        maxWidth: theme.custom.contentAreaWidth,
+    },
+    mainTitle: {
+        paddingLeft: 0,
+    },
+    FormControl: {
+        padding: `0 0 0 ${theme.spacing(1)}px`,
+        width: '100%',
+        marginTop: 0,
+    },
+    FormControlOdd: {
+        padding: `0 0 0 ${theme.spacing(1)}px`,
+        backgroundColor: theme.palette.background.paper,
+        width: '100%',
+        marginTop: 0,
+    },
+    FormControlLabel: {
+        marginBottom: theme.spacing(1),
+        marginTop: theme.spacing(1),
+        fontSize: theme.typography.caption.fontSize,
+    },
+    buttonSection: {
+        paddingTop: theme.spacing(3),
+    },
+    saveButton: {
+        marginRight: theme.spacing(1),
+    },
+    helpText: {
+        color: theme.palette.text.hint,
+        marginTop: theme.spacing(1),
+    },
+    extraPadding: {
+        paddingLeft: theme.spacing(2),
+    },
+    addNewOther: {
+        paddingTop: 40,
+    },
+    titleGrid: {
+        ' & .MuiGrid-item': {
+            padding: 0,
+            margin: 0,
+        },
+    },
+    descriptionForm: {
+        marginTop: theme.spacing(1),
+    },
+    progress: {
+        marginLeft: theme.spacing(1),
+    },
+});
+
+let apiCategory = {};
+let mhistory;
+
+const updateAPICategory = (
+    apiCategoryId,
+    name,
+    setUpdatingAPICategory,
+    intl,
+) => {
+    console.log(apiCategoryId, name, apiCategory['description']);
```

Comment:
is this needed?

Reply:
That line is removed with commit: https://github.com/wso2/carbon-apimgt/pull/8429/commits/26d9169b808a61521bcf811b038010b581484e75